src/server/game/Spells/Auras/SpellAuras.cpp

```diff
@@ -2636,6 +2637,7 @@ void UnitAura::FillTargetMap(std::map<Unit*, uint8> & targets, Unit* caster)
                         break;
                     }
                     case SPELL_EFFECT_APPLY_AREA_AURA_PET:
+                        #pragma GCC diagnostic ignored "-Wimplicit-fallthrough"
```

Comment:
wow wow wow, stop, please and loot this https://en.cppreference.com/w/cpp/language/attributes/fallthrough

Reply:
@Winfidonarleyan yes I will add this for sure :)

Reply:
Okay